### DISCUSSION TITLE
FIxed Wing Motor Not Responding to Input from Controller

### DISCUSSION THREAD

**Discussion by SuboptimalOutcome** [iNavFlight/inav#7252] [Q&A]
I'm a new hobbyist in the process of building my first fixed wing drone. I am building a Sonic Modell SkyHunter Racing drone with the recommended servo/motor/ESC kit. The ESC is a FlyColor FlyDragon lite 30A. I am using a Matek F765 FC with INAV 3.0 running. In the mixer in INAV I have set up the drone as an airplane without a rudder. I have signal to my receiver showing up in INAV with the controller on, and I have all my servos functioning properly. I can throttle up my motor using INAV in the outputs tab. I'm using a FrSky Taranis X9 Lite Controller with a FrSky R9 MX receiver. I have my channel map set as TAER in INAV and on my controller and the throttle seems to be receiving signal in INAV from the controller, with all the channels responding correctly. I set the throttle range from 1000-2000 in INAV and on the controller. I cannot get the motor to respond to the controller. I have read that manual ESC calibration with the controller might be the issue but trying the calibration as suggested has not worked so far and I have run out of ideas on how to get the motor working at this point. I appreciate any tips or suggestions with this problem!

**Comment by breadoven** [comment]
Armed ?

I'm guessing the ESC protocol is set to standard  since the motor runs in the output tab ?

https://inavfixedwinggroup.com/guides/getting-started/inav-basic-troubleshooting-guide/#1-the-flight-controller-arms-but-motor-wont-spin

**Reply by SuboptimalOutcome** [reply]
Wow thanks for that, I guess that was the problem. Or at least part of it.  I hadn't set an arm switch and the motor wasn't armed.😂 I set an arm switch and hooked up a GPS so the all the pre-arm checks were green and then it finally armed and for a brief period the throttle was working. I still noticed that the throttle range wasn't quite right since it took almost a third of the stick rotation to get the motor spinning. I now have another problem because now even though it's arming, the throttle is again not responding to the controller and I'm not sure what changed. It worked briefly and then I disarmed it and re-armed and when I tried the throttle again, it no longer worked.

**Reply by paul-fornage** [reply]
Make sure you are actually arming. There are several factors that could prevent the board from arming despite the switch being flipped. If you have an FPV system with OSD, you can watch the video feed, and it will display the reason for not arming. If it does successfully arm, you can preview what the board is telling the ESC to do in the outputs tab. (edit:) This will tell you which component is not working as you intended; the ESC or the FC-(flight controller)

**Comment by Dafics** [comment]
Same issue for me. Equipment slightly different but effectively the same.
Also new to OpenTx (with TX16s) but the configurator Mode page indicates setup properly for arming but the arming does not happen (an setup page) despite all green ticks. I am not getting video from the system so I will check "status" in the CLI.

**Reply by Dafics** [reply]
Arming fixed. CLI Status pointed to VTX. Found a bent pin and fixed.
Still no video despite CLI indicating VTX ok.
